templates/ocs-deployment/Pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{pod_name | default("nginx1") }}
+  namespace: {{ project_name | default("default")}}
+spec:
+  containers:
+   - name: web-server
+     image: nginx
+     volumeMounts:
+       - name: mypvc
+         mountPath: /var/lib/www/html
+  volumes:
+   - name: mypvc
+     persistentVolumeClaim:
+       claimName: {{ pvc_name | default ("test-rbd-device")}}
+       readOnly: false
```

Comment:
add new line at the end of the file, apply to all

Reply:
Can we change the file name to pod_nginx.yaml or app_pod_nginx.yaml or nginx_app_pod.yaml